calculator.py

```diff
@@ -0,0 +1,35 @@
+import datetime
+ 
+all_data = {
+    "Transaction Type": "",
+    "Origin or Destination": "",
+    "Amount": "",
+    "Date": ""
```

Comment:
all your dictionary keys should be lowercase with underscores between two or more words